Library/Formula/libodb-qt.rb

```diff
@@ -0,0 +1,78 @@
+class LibodbQt < Formula
+  homepage "http://www.codesynthesis.com/products/odb/"
+  url "http://www.codesynthesis.com/download/odb/2.4/libodb-qt-2.4.0.tar.gz"
+  sha256 "f48c563f653df178866301e4c5069b09b697d98f54dcc9ef8714157002156183"
+
+  option "without-qt", "Don't install the Qt library"
+  option "with-qt5", "Install version 5 of the framework instead of 4."
+
+  depends_on "libodb"
+  if build.without?("qt")
+  elsif build.with?("qt5")
+    depends_on "qt5"
+  else
+    depends_on "qt"
+  end
+
+  def install
+    qmake = "#{ENV["QMAKE"]}"
+
+    if qmake.empty?
+      qmake = `which qmake`.strip
+      if qmake.empty?
+        onoe "    Qt is a dependency, but you chose \"--without-qt\".
+             You need to indicate \`qmake\` path to homebrew as:
+             QMAKE=~/Qt/5.4/clang_64/bin/qmake brew install libodb-qt".undent
+        exit 1
+      end
+    end
```

Comment:
Most Homebrew users do not install multiple versions of Qt. Again, just add `depends_on "qt"` or move this to your own tap, thanks. I don't really want to discuss the Qt issue any more.